pyartcd/pyartcd/pipelines/ocp4_konflux.py

```diff
@@ -209,6 +214,27 @@ async def clean_up(self):
             self.runtime.cleanup_sources('konflux_build_sources'),
         ])
 
+    def trigger_bundle_build(self):
+        record_log_path = Path(self.runtime.doozer_working, 'record.log')
+        if not record_log_path.exists():
+            LOGGER.warning('record.log not found, skipping bundle build')
+            return
+        with record_log_path.open('r') as file:
+            record_log: dict = record_util.parse_record_log(file)
+        records = record_log.get('image_build_konflux', [])
+        operator_nvrs = []
+        for record in records:
+            if record['has_olm_bundle'] == '1' and record['status'] == '0' and record.get('nvrs', None):
+                operator_nvrs.append(record['nvrs'].split(',')[0])
+        if operator_nvrs:
+            jenkins.start_olm_bundle_konflux(
```

Comment:
Do we first need an aos-cd-jobs PR to define the new olm-bundle-konflux job?

Reply:
looks like we can merge as `--skip-bundle-build` is True by default

Reply:
I've updated the PR to set `--skip-bundle-build` by default until we believe this feature becomes stable.